src/archive/Extract.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
Is there a reason why you're keeping `src/archive`?  We should always only have one version of a given Python file at a given point in Git history.  Make sense?

Reply:
Makes perfect sense. Thought I'd removed archive before committing... I think making an archive folder is a bad habit I can afford to break once I get a little more comfortable with GitHub. I do understand why using an 'archive' is not the right way of doing things - it is just something I subconsciously picked up from work, where GitHub isn't used.